### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by kmalta**
R.....
E.....
F.....
A....
C....
T.....
O....
R.....
....REFACTOR!!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ const app = express();
 app.listen(3000, () => console.log('Example app listening on port 3000!'));
 app.use(express.static(path.join(__dirname, '/public')));
 
-app.use('/spotify', require('./routes/spotify'));
+app.use('/spotify', require('./routes/routes'));
```
Comment (trevorfrese) [inline index.js]: This is a bad file name, please revert it back to spotify. The routes are still routes that relate to spotify

**Review by trevorfrese — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (kmalta) [reply index.js]: I was going to make it a general routes file, but that's probably better.  Changing it!

**Comment by kmalta** [comment]
It has been changed.

**Review by trevorfrese — APPROVED** [review]
LGTM

**Pull request merged**